.commitlintrc.ts

```diff
@@ -0,0 +1,10 @@
+import type { UserConfig } from "@commitlint/types";
+
+const Configuration: UserConfig = {
+  extends: ["@commitlint/config-conventional"],
+  formatter: "@commitlint/format",
+  parserPreset: "conventional-changelog-conventionalcommits",
+  helpUrl: `Please refer to the 'Conventional Commits' section in the README or search for 'Conventional Commits' on Google for more information. Here is popular cheat sheet: https://kapeli.com/cheat_sheets/Conventional_Commits.docset/Contents/Resources/Documents/index`,
```

Comment:
```suggestion
  helpUrl: `Please refer to the 'Conventional Commits' section in the README.
    You can look at 'Conventional Commits' official docs: https://www.conventionalcommits.org/en/
    Here is popular cheat sheet: https://kapeli.com/cheat_sheets/Conventional_Commits.docset/Contents/Resources/Documents/index`,
```

Looks a bit better with new lines and tabs.